test/scenarios/move_overwriting_dir/scenario.js

```diff
@@ -0,0 +1,53 @@
+/* @flow */
+
+/*:: import type { Scenario } from '..' */
+
+module.exports = ({
+  noremote: true,
+  init: [
+    { ino: 1, path: 'dst/' },
+    { ino: 2, path: 'dst/dir/' },
+    { ino: 3, path: 'dst/dir/subdir/' },
+    { ino: 4, path: 'dst/dir/file', content: 'overwriten' },
+    { ino: 5, path: 'dst/dir/file3', content: 'not-overwriten' },
+    { ino: 6, path: 'dst/dir/subdir/file', content: 'sub-overwriten' },
+    { ino: 7, path: 'dst/dir/subdir/file5', content: 'sub-not-overwriten' },
+    { ino: 8, path: 'src/' },
+    { ino: 9, path: 'src/dir/' },
+    { ino: 10, path: 'src/dir/subdir/' },
+    { ino: 11, path: 'src/dir/file', content: 'overwriter' },
```

Comment:
`overwriter` & `overwriten` are hard to distinguish (should actually be `overwritten`).
What about `old file content` & `new file content`?